test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/NodeEndToEndTests.cs

```diff
@@ -193,7 +193,7 @@ public async Task Scenario_Logging()
             {
                 userLogs = Fixture.Host.GetScriptHostLogMessages(userCategory).Select(p => p.FormattedMessage).ToList();
                 consoleLog = Fixture.Host.GetScriptHostLogMessages(WorkerConstants.FunctionConsoleLogCategoryName).Select(p => p.FormattedMessage).SingleOrDefault();
-                return userLogs.Count == 10 && consoleLog != null;
+                return userLogs.Count == 11 && consoleLog != null;
```

Comment:
The worker had a minor behavior change around logs, which explains why this test was failing. See discussion here if you're curious: https://github.com/Azure/azure-functions-nodejs-worker/pull/547#discussion_r822203746